build/cygnus.js

```diff
@@ -1,29 +1,28 @@
-var _this2 = this,
-    _arguments = arguments;
-
 /* global cygnus, Blob, Worker, XMLHttpRequest */
 
+const _arguments = arguments;
+
 module.exports = {
   supportsHistory: !!window.history,
   supportsWorkers: !!window.Worker,
   supportsPromises: !!Promise,
   ready: false,
   pages: {},
-  init: () => {
+  init: function () {
+    if (!this.ready) {
+      window.cygnus = this;
+      window.onpopstate = this.handlePopState;
```

Comment:
Attaching to window is not usually the method people expect with commonjs. Generally attaching all imports to a global object isn't advisable, it's better to use a module loader (like commonjs, amd, etc). I might make it such that attaching to window is optional based on an argument passed to `init` -- most people installing through npm will be using something like browserify or webpack or systemjs etc. and will not want it to be attached to `window`.